charts/renku/requirements.yaml

```diff
@@ -6,7 +6,7 @@ dependencies:
 - name: renku-ui
   alias: ui
   repository: "https://swissdatasciencecenter.github.io/helm-charts/"
-  version: 0.11.0-a6fe6fe
+  version: 0.11.0-f9871b1
```

Comment:
if you use this chart, you may want to merge the branch `703ui-integration-tests` that contains the relevant changes for the integration test

Reply:
ah great, thanks, will do